sheet/ozymandias/Skills.qml

```diff
@@ -0,0 +1,204 @@
+import QtQuick 2.0
+import org.lasath.psychic_bear 1.0
+import ".."
+
+Item {
```

Comment:
You can get the total number of unspent skill points now if you want.
All you have to do is add an extra attribute to your character with the amount of total skill points you have, like [this](https://github.com/shocklateboy92/psychic-bear/blob/6279606b7eb9f11a4742b2d56a008b7d57b2553c/sheet/fernie/Character.qml#L218-L236).